docs-ui/stories/assets/icons/popper.tsx

```diff
@@ -29,31 +55,27 @@ const IconPopper = ({icon, setSelectedIcon, boxRef}: Props) => {
   const [isCircled, setIsCircled] = useState(icon.defaultProps?.isCircled ?? false);
   const [isSolid, setIsSolid] = useState(icon.defaultProps?.isSolid ?? false);
 
-  /**
-   * Generate and update code sample based on prop states
-   */
-  const getCodeSample = () => {
-    return `<Icon${icon.name} color="gray500" size="${size}"${
-      isCircled ? ' isCircled' : ' '
-    }${isSolid ? ' isSolid' : ' '} />`;
-  };
-  const [codeSample, setCodeSample] = useState(getCodeSample());
```

Comment:
the default state and setCodeSample on useEffect indicates that this should probably have been a useMemo